.travis.yml

```diff
@@ -46,6 +50,13 @@ matrix:
       osx_image: xcode10
       sudo: required
       env: SWIFT_SNAPSHOT=4.2
+    - os: osx
+      osx_image: xcode10
+      sudo: required
+      env: SWIFT_SNAPSHOT=4.2 KITURA_NIO=1
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew install libressl ; fi
```

Comment:
I guess this is going to be required for all Kitura packages on macOS? If so, perhaps it would be better added to Package-Builder.

Reply:
Sure, will do